src/tools/perf/lib/libperf.c

```diff
@@ -1741,3 +1742,13 @@ size_t ucx_perf_get_message_size(const ucx_perf_params_t *params)
 
     return length;
 }
+
+unsigned rte_peer_index(unsigned group_size, unsigned group_index)
+{
+    int size       = group_size;
+    int index      = group_index;
```

Comment:
pls do not use "index" as variable name as it conflicts with glibc function name